tests/test_core.py

```diff
@@ -97,9 +97,13 @@ def test_init_df_with_extra_col(test_pd_df):
 
     df = IamDataFrame(tdf)
 
+    # check that extra-cols attribute is set correctly
     assert df.extra_cols == [extra_col]
-    pd.testing.assert_frame_equal(df.timeseries().reset_index(),
-                                  tdf, check_like=True)
+
+    # check that timeseries data is as expected
```

Comment:
Is this change related to the PR? Could not figure out why.

Reply:
As with the other comment, this resolves a pandas v1.2 issue on the fly.